app/src/main/java/chess/Board.java

```diff
@@ -3,23 +3,74 @@
 import chess.pieces.Pawn;
 
 import java.util.ArrayList;
+import java.util.Collections;
 import java.util.List;
+import java.util.Optional;
+import java.util.function.Function;
 
 public class Board {
+    private List<List<Pawn>> chessBoard;
     private List<Pawn> pawns;
+    private final int CHESS_BOARD_START_INDEX = 0;
+    private final int CHESS_BOARD_END_INDEX = 8;
 
     public Board(){
+        this.chessBoard = new ArrayList<>();
         this.pawns = new ArrayList<>();
     }
 
+    public void initialize(){
+        final int BLACK_PAWN_ROW = 1;
+        final int WHITE_PAWN_ROW = 6;
+
+        for (int i = CHESS_BOARD_START_INDEX; i < CHESS_BOARD_END_INDEX; i++){
+          if(i == BLACK_PAWN_ROW){
+              addPawnRow("black");
+          }
+          else if(i == WHITE_PAWN_ROW){
+              addPawnRow("white");
+          } else {
+              addEmptyRow();
+          }
+        }
+    }
+
+    private void addPawnRow(String color){
+        List<Pawn> pawnRow = new ArrayList<>();
+        for (int i = CHESS_BOARD_START_INDEX; i < CHESS_BOARD_END_INDEX; i++){
+            pawnRow.add(new Pawn(color));
+        }
+        chessBoard.add(pawnRow);
+    }
+
+    private void addEmptyRow(){
+        final int LENGTH_OF_ROW = 8;
+        chessBoard.add(new ArrayList<>(Collections.nCopies(LENGTH_OF_ROW, null)));
```

Comment:
```suggestion
        chessBoard.add(new ArrayList<>(L
```
```suggestion
        chessBoard.add(new ArrayList<>(LENGTH_OF_ROW);
```

null은 넣는 게 아니라서 아무 것도 안 넣으면 null 이 들어갑니다.
null은 가능하면 로직에 활용하지 않는 게 좋구요. 

꼭 쓰고 싶다면 Optional 을 활용하는 것도 방법입니다.
저라면 현재 Pawn 에 empty 상태를 하나 추가하는 방법을 선택했을 것 같아요.